src/ralph/back_office/models.py

```diff
@@ -27,6 +27,10 @@ class Warehouse(NamedMixin, TimeStampMixin, models.Model):
     pass
 
 
+class AssetOwner(NamedMixin.NonUnique, TimeStampMixin, models.Model):
```

Comment:
it is confusing with user/owner of the asset - IMO we should rename it. @vi4m any idea how to name it?